.claude/skills/git/context/branch-strategies.md

```
@@ -0,0 +1,347 @@
+### Convert to Feature Branch
+
+```bash
+# If working on main by mistake
+git branch feature/123-description  # Create branch
+git reset --hard origin/main        # Reset main
+git checkout feature/123-description # Switch to feature
+```
+
+## Branch Lifecycle
+
+### 1. Creation
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)